pkg/kubelet/network/dns/dns.go

```diff
@@ -209,6 +210,7 @@ func parseResolvConf(reader io.Reader) (nameservers []string, searches []string,
 	// Each option is recorded as an element in the array.
 	options = []string{}
 
+	allErrors := []error{}
```

Comment:
var allErrors []error

Reply:
curious why is this better ? ( i am only changing it )